lib/calibex.ex

```diff
@@ -38,9 +57,15 @@ defmodule Calibex do
   ## Example usage : email event request generation 
 
   ```
-  Calibex.request(dtstart: Timex.now, dtend: Timex.shift(Timex.now,hours: 1), summary: "Mon évènement",
-            organizer: "arnaud.wetzel@example.com", attendee: "jeanpierre@yahoo.fr", attendee: "jean@ya.fr")
-   |> Calibex.encode
+  Calibex.request(
+    dtstart: Timex.now(),
+    dtend: Timex.shift(Timex.now(), hours: 1),
```

Comment:
Can this example be converted to DateTime, eventually ?

Reply:
Good catch :+1: